os/drivers/audio/alc5658.c

```diff
@@ -985,10 +986,8 @@ static int alc5658_ioctl(FAR struct audio_lowerhalf_s *dev, int cmd, unsigned lo
 #endif
 	}
 	break;
-
-
 	default:
-		audvdbg("Ignored\n");
+		auddbg("alc5658_ioctl received unkown cmd 0x%x\n", cmd);
```

Comment:
good change for information. but verbose is correct, I think.

Reply:
But unknown cmd is a error.

Reply:
No, it is not error case, it just undefined case for specific file operation(in this case audio operation of alc5658)
Any arg value can be sent to driver layer, so tizenrt usually using verbose for default value.